codebase-pipelines/tests/unit.tftest.hcl

```
@@ -737,7 +756,25 @@ run "test_iam_documents" {
     error_message = "Should be: Allow"
   }
   assert {
-    condition     = one(data.aws_iam_policy_document.ecr_access_for_codebase_pipeline.statement[0].actions) == "ecr:DescribeImages"
+    condition = data.aws_iam_policy_document.ecr_access_for_codebase_pipeline.statement[0].actions == toset([
+      "ecr:DescribeImages",
+      "ecr:BatchGetImage",
+      "ecr:GetDownloadUrlForLayer"
+    ])
+    error_message = "Unexpected actions"
+  }
+
+  # SSM access
+  assert {
+    condition     = data.aws_iam_policy_document.deploy_ssm_access.statement[0].effect == "Allow"
+    error_message = "Should be: Allow"
+  }
+
+  assert {
+    condition = data.aws_iam_policy_document.deploy_ssm_access.statement[0].actions == toset([
+      "ssm:GetParameter",
+      "ssm:GetParameters"
+    ])
     error_message = "Unexpected actions"
   }
 }
```

You can also test the resource name here
```suggestion
}
  assert {
    condition = one(data.aws_iam_policy_document.deploy_ssm_access.statement[0].resources) == "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter/codebuild/slack_*"
    error_message = "Unexpected resources"
  }
```

---

Ah hadn't seen that thank you